lib/rmq.service.ts

```diff
@@ -107,7 +107,7 @@ export class RMQService {
 			const timerId = setTimeout(() => {
 				reject(new RMQError(`${ERROR_TIMEOUT}: ${timeout}`, ERROR_TYPE.TRANSPORT));
 			}, timeout);
-			this.sendResponseEmitter.once(correlationId, (msg: Message) => {
+			this.sendResponseEmitter.on(correlationId, (msg: Message) => {
```

Comment:
Why do we need `on` vs `once`? Will it cause memory leak?

Reply:
That's my bad, I was testing other stuff earlier, will change that